schema/ingestion_config/v1.0.0/ingestion_config_models.yaml

```diff
@@ -688,12 +688,14 @@ classes:
       value:
         description: The value for the voxel spacing literal.
         multivalued: true
-        range: float
         required: true
-        # TODO: uncomment after any_of is fixed https://github.com/linkml/linkml/issues/1521
-        # any_of:
-        #   - range: float
-        #   - range: FloatFormattedString
+        range: float
+        any_of:
```

Comment:
should the any_of also be commented out?

Reply:
Oop yes, let me fix that right now